process.c

```diff
@@ -624,18 +628,25 @@ rb_last_status_get(void)
 
 /*
  *  call-seq:
- *     Process.last_status   -> Process::Status or nil
+ *    Process.last_status -> Process::Status or nil
+ *
+ *  Returns a Process::Status object representing the most recently exited
+ *  child process in the current thread, or +nil+ if none:
  *
- *  Returns the status of the last executed child process in the
- *  current thread.
+ *    Process.spawn('ruby', '-e', 'exit 13')
+ *    Process.wait
+ *    Process.last_status # => #<Process::Status: pid 14396 exit 13>
  *
- *     Process.wait Process.spawn("ruby", "-e", "exit 13")
- *     Process.last_status   #=> #<Process::Status: pid 4825 exit 13>
+ *    Process.spawn('ruby', '-e', 'exit 14')
+ *    Process.wait
+ *    Process.last_status # => #<Process::Status: pid 4692 exit 14>
  *
- *  If no child process has ever been executed in the current
- *  thread, this returns +nil+.
+ *    Process.spawn('ruby', '-e', 'exit 15')
+ *    # 'exit 15' has not exited.
```

Comment:
The `last_status` is set by `Process.wait`, or not changed until `wait`.
```suggestion
 *    # 'exit 15' has not been reaped by #wait.
```

Reply:
Fixed.